src/Editor/src/Scripting/PythonInterface.cpp

```
@@ -125,11 +132,36 @@ static PyObject *osre_project_close(PyObject*, PyObject*, PyObject*) {
 
+PyDoc_STRVAR(osre_project_import_doc, "ToDo.");
+static PyObject *osre_project_import(PyObject *, PyObject *args, PyObject *keywds) {
+    static char *kwlist[] = { "asset_name", NULL };
+    char *asset_name = nullptr;
+    if (!PyArg_ParseTupleAndKeywords(args, keywds, "s", kwlist, &asset_name)) {
+        return nullptr;
+    }
+
+    if (gActiceProject == nullptr) {
+        return nullptr;
+    }
```

Cppcheck (2.10-2)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)